src/filter/facebl0r/facebl0r.cpp

```diff
@@ -16,8 +16,8 @@
 
 
 #include <opencv2/core/version.hpp>
-#define CV_VERSION_NUM (CV_VERSION_MAJOR * 10000 \
-                      + CV_VERSION_MINOR * 100 \
+#define CV_VERSION_NUM (CV_MAJOR_VERSION * 10000 \
+                      + CV_MINOR_VERSION * 100 \
```

Comment:
This does not look needed or even preferred according to
https://docs.opencv.org/master/d7/dad/version_8hpp.html

Reply:
Since  last change e1ecb144ca58cb4d68ef326d0ae224025a171fef, build is broken with opencv 2.4.9.1 installed (Stretch) 

I have take a look at both v2 & v3 version.hpp to understood the problem, and for me it's the easier solution.

@ddennedy  Does me fix broke your (openvc3) build ?

Reply:
OK, I agree to the change for opencv version check.